jekyll/_cci2/orb-author.md

```diff
@@ -17,104 +17,125 @@ version:
     ```shell
     git push origin <default-branch>
     ```
     Once complete, return to your terminal and confirm the changes have been pushed.
 
-1. **Complete and write your orb.**
-<br>
-    The CLI will finish by automatically following your new orb project on CircleCI and generating the first development version `<namespace>/<orb>@dev:alpha` for testing (a hello-world sample).
+1. **Complete the setup**
 
-    You will be provided with a link to the project building on CircleCI where you can view the validation, packing, testing, and publication process.
-    You should also see the CLI has automatically migrated you into a new development branch, named `alpha`.
+    Once the changes have been pushed, return to your terminal and continue the setup process. The CLI will now automatically follow the project on CircleCI, and attempt to trigger a pipeline to build and test your orb with sample code.
 
-    From your new branch you are now ready to make and push changes. From this point on, on every commit, your orb will be packed, validated, tested (optional), and can be published.
+    You will be provided with a link to the project building on CircleCI where you can view the full pipeline.
+    You should also see the CLI has automatically migrated you into a new development branch, named `alpha`. You can use any branch naming you would like, you do not need to exclusively develop on `alpha`.
 
-    When you are ready to deploy the first major version of your orb, find information on deploying changes with semver versioning in the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs/) guide.
+    Because we are making use of [dynamic configuration]({{site.baseurl}}/2.0/dynamic-config/), we must first enable this feature. You will receive an error on your first pipeline that will state that this feature is not enabled.
+
+    Following the [Getting started with dynamic config in CircleCI]({{site.baseurl}}/2.0/dynamic-config/#getting-started-with-dynamic-config-in-circleci), open the **Project Settings** page for your orb on CircleCI, navigate to the **Advanced** tab, and click on the **Enable dynamic config using setup workflows** button.
+
+1. **Develop your orb**
+
+    From a non-default branch (you will be moved to the `alpha` branch automatically at setup), begin modifying the sample orb code to your liking. On each _push_, your orb will be automatically built and tested.
+
+    Be sure to view the _[.circleci/test-deploy](https://github.com/CircleCI-Public/Orb-Template/blob/main/.circleci/test-deploy.yml)_ file to view how your orb components are being tested, and modify your tests as you change your orb. Learn more about testing your orb in the [Orb Testing documentation]({{site.baseurl}}/2.0/testing-orbs/).
+
+    When you are ready to deploy the first production version of your orb, find information on deploying changes in the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs/) guide.
 
 ### Writing your orb
```

Comment:
The following sections - I'm not sure why they are under the main Orb Development Kit heading. Also who are these sections for - are they reference material for people currently writing an orb? Or like a deep dive into the structure of the packaged orb?

Reply:
Maybe we change the section name: "Create, test and publish an orb" and then add a note at the top that the section uses the Orb dev kit

Reply:
I'll close this thread so we can get this merged and revisit later

Reply:
I am slightly concerned about losing the heading link for the dev kit, but it is at the top of the page so should be ok. Sending a commit